python/ray/workflow/api.py

```diff
@@ -485,6 +485,7 @@ def wait(
         A list of ready workflow results that are ready and a list of the
         remaining workflows.
     """
+    raise NotImplementedError("workflow.wait() is temporarily disabled.")
```

Comment:
I'm with with this for short term. But since this is a public API, we should fix this ASAP.

Reply:
Later I decide to remove `wait` temporarily in this PR.